src/js/utils/language.js

```diff
@@ -35,3 +36,16 @@ export function getLabel(language) {
 export function getCode(language) {
     return langToCode[language] || '';
 }
+
+export function getPlayerLanguage() {
```

Comment:
Wouldn't `getPageLanguage` be a more appropriate name. We will make this the language that the player uses, but we're getting it from the page.